src/cert-enroll/cert-install_lighttpd

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Install the generated key and certificate as TLS credentials for a web server
+# based based on the lighttpd daemon.
+#
+# Copyright (C) 2023 Andreas Steffen
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+##############################################################################
+# Change into the certificates directory
+#
+cd $CERTDIR
+
+##############################################################################
+# Install the web server's TLS key and certificate in single file
+#
+cat $HOSTKEY $HOSTCERT > /etc/lighttpd/https-cert.pem
+
+##############################################################################
+# Restart the lighttpd daemon 
+#
+/etc/init.d/lighttpd restart || exit 1
```

Comment:
Ok, so probably `USE_CERT_ENROLL_TIMER` could be used to differentiate between `init.d/<daemon> restart` and `systemctl restart <daemon>`.

Reply:
Ok, so probably `USE_CERT_ENROLL_TIMER` could be used to differentiate between `init.d/<daemon> restart` and `systemctl restart <daemon>`.

Reply:
Maybe, although users might not enable the timer option even if systemd is used on the system.

I'm not sure how to handle this this correctly. Maybe use systemctl if the init script doesn't exist (although it might exist as fallback)? Or check if `systemctl is-active lighttpd.service` returns successfully (although that could also indicate it exists but is just currently not running)? I guess checking if the unit exists could be done via `systemctl list-unit-files lighttpd.service`?

Reply:
We check for `systemctl list-unit-files lighttpd.service`